bin/local-env/install-wordpress.sh

```diff
@@ -17,15 +17,26 @@ dc up -d >/dev/null 2>&1
 # Get the host port for the WordPress container.
 HOST_PORT=$(dc port $CONTAINER 80 | awk -F : '{printf $2}')
 
-# Wait until the Docker containers are running and the WordPress site is
-# responding to requests.
+# Wait until the WordPress site is responding to requests.
 echo -en $(status_message "Attempting to connect to WordPress...")
 until $(curl -L http://localhost:$HOST_PORT -so - 2>&1 | grep -q "WordPress"); do
     echo -n '.'
     sleep 5
 done
 echo ''
 
+# Wait until the database container is ready.
+echo -en $(status_message "Waiting for database connection...")
+until $(container bash -c "echo -n > /dev/tcp/mysql/3306" >/dev/null 2>&1); do
+    echo -n '.'
+    sleep 5
+done
+echo ''
+
+# Create the database if it doesn't exist.
+echo -e $(status_message "Creating the database (if it does not exist)...")
+mysql -e 'CREATE DATABASE IF NOT EXISTS wordpress;'
```

Comment:
Shouldn't this use `WORDPRESS_DB_NAME` from the env file?

Reply:
Yes it should have, I'll make the amendment.